org.civicrm.reqangular/src/common/services/file-upload.js

```diff
@@ -65,9 +65,28 @@ define([
             formData: [{
               entity_table: customSettings.entityTable || error('entityTable'),
               crm_attachment_token: customSettings.crmAttachmentToken || error('crmAttachmentToken')
+            }],
+            filters: [{
+              name: 'fileFormatFilter',
+              fn: function (item /*{File|FileLikeObject}*/ , options) {
+                var type = '|' + item.type.slice(item.type.lastIndexOf('/') + 1) + '|';
```

Comment:
also why do you need to wrap the `type` in "|"s? Isn't it sufficient to use the type to look in the list? why are the "|"s necessary for the search?